eng/native/init-distro-rid.sh

```diff
@@ -41,7 +41,7 @@ initNonPortableDistroRid()
             # We have forced __PortableBuild=0. This is because -portablebuld
             # has been passed as false.
             if (( isPortable == 0 )); then
-                if [[ "${ID}" == "rhel" || "${ID}" == "rocky" ]]; then
+                if [[ "${ID}" == "rhel" || "${ID}" == "rocky" || "${ID}" == "almalinux" ]]; then
                     # remove the last version digit
                     VERSION_ID="${VERSION_ID%.*}"
```

Comment:
Instead of accounting for each derivative distro separately, how about:
```suggestion
                # for RHEL and its derivatives, remove the last version digit (if present)
                if [[ "${ID}" == "rhel" || "${ID_LIKE}" =~ .*rhel.* ]]; then
                    VERSION_ID="${VERSION_ID%.*}"
```